Update README.md (#427)
SamSchott/maestral: README.md

## Installation

> An app bundle is provided for macOS High Sierra and higher and can be downloaded from the
> Releases tab. This app Bundle is also package as a Homebrew cask.

Reviewer: ```suggestion
An app bundle is provided for macOS High Sierra and higher. It can be downloaded from the
[Releases](https://github.com/SamSchott/maestral/releases) tab and is also packaged as a Homebrew cask.
```

I would restructure those sentences to have the download information bundled in one sentence and the packaging format in another.

> On other platforms, you can download and install Maestral as a Python PyPI package, as a Docker package.

Reviewer: ```suggestion
On other platforms, you can download and install Maestral as a Python package from PyPI or as a Docker image from Docker Hub.
```

Its not a "PyPI package" but rather a Python package which happens to be hosted on PyPI...

For more detailed information on the installation, setup and system requirements, please

> check the [documentation](https://samschott.github.io/maestral/docs/installation).

Reviewer: ```suggestion
check the [documentation](https://maestral.app/docs/installation).
```
We now have our own domain :)

### Homebrew

> The official Maestral releases are also available as Homebrew casks. If you have [Homebrew](https://brew.sh) on your system, you can install using:

Reviewer: Check line length. I've been sticking to 90 chars.
